lib/v2/nikkei-asia/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'asia.nikkei.com': {
+        _name: 'Nikkei Asia',
+        '.': [
```

Comment:
```suggestion
    'nikkei.com': {
        _name: 'Nikkei Asia',
        'asia': [
```

Reply:
@TonyRL Nikkei Asia (asia.nikkei.com) and Nikkei (nikkei.com) are two separate publications and websites, owned by the same owner.